Add instructions about the secretfile permissions (#11952)
DataDog/integrations-core: varnish/README.md

sudo usermod -G varnish -a dd-agent
```

> If you are going to be using a `secretfile` you need to ensure it is readable by the `dd-agent` user.

Reviewer: ```suggestion
If you use a `secretfile`, you must ensure it is readable by the `dd-agent` user.
```
